README.md

```diff
@@ -1,5 +1,13 @@
 # Main Street Brewing Rewards Program 🍺
 
+## Technology Used
+
+- React
+- React Native
+- Xcode
+- GraphQL
+  - GraphCool
```

Comment:
GraphCool is not part of GraphQL. 
React Native includes React, no need to mention it.
You are not using XCode for any dev purposes.

Reply:
ok il make changes